src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/main.js

```diff
@@ -0,0 +1,243 @@
+                                name: 'published',
+                                content: this.translations.published,
+                                type: 'checkbox_readonly'
+                            },
+                            {
+                                attribute: 'customUrl',
+                                name: 'customUrl',
+                                content: this.translations.customUrl
+                            },
+                            {
+                                attribute: 'targetTitle',
+                                name: 'targetTitle',
+                                content: this.translations.targetTitle,
+                                type: function(content) {
+                                    if (content === '') {
+                                        return this.translations.noTarget;
+                                    }
+
+                                    return content;
+                                }.bind(this)
+                            },
+                            {
+                                attribute: 'changed',
+                                name: 'changed',
+                                content: this.translations.changed,
+                                type: 'datetime'
+                            },
+                            {
+                                attribute: 'changerFullName',
+                                name: 'changerFullName',
+                                content: this.translations.changer
+                            },
+                            {
+                                attribute: 'created',
+                                name: 'created',
+                                content: this.translations.created,
+                                type: 'datetime'
+                            },
+                            {
+                                attribute: 'creatorFullName',
+                                name: 'creatorFullName',
+                                content: this.translations.creator
+                            }
+                        ]
+                    }
+                }
+            ]);
+        },
+
+        /**
+         * Start edit overlay for given id.
+         *
+         * @param {Integer} id
+         */
+        edit: function(id) {
+            this.sandbox.start([
+                {
+                    name: 'webspace/settings/custom-url/overlay@sulucustomurl',
+                    options: {
+                        el: '#webspace-custom-url-form-overlay',
+                        id: id,
+                        webspace: this.data,
+                        saveCallback: this.save.bind(this),
+                        translations: this.translations
+                    }
+                }
+            ]);
+        },
+
+        /**
+         * Deletes selected records from datagrid after asking for confirmation.
+         */
+        delete: function() {
```

Comment:
Didn't you encounter any problem with that key? I thought we named that function `del` in other places, because `delete` is a keyword.